src/main/webapp/charts.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Movie Information Charts</title>
+    <script type="text/javascript" src="https://www.gstatic.com/charts/loader.js"></script>
+    <script type="text/javascript">
```

Comment:
optional: break the JS into a separate file

Reply:
Refactoring done.